pkgs/development/python-modules/aioresponses/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, python
+, pbr
+, aiohttp
+, ddt
+, asynctest
+, pytest_3
+}:
+
+buildPythonPackage rec {
+  pname = "aioresponses";
+  version = "0.6.0";
+  disabled = !(pythonAtLeast "3.5");
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ii1jiwb8qa2y8cqa1zqn7mjax9l8bpf16k4clv616mxw1l0bvs6";
+  };
+
+  buildInputs = [ pbr aiohttp ddt asynctest pytest_3 ];
```

```suggestion
  nativeBuildInputs = [ 
    pbr
  ];
  
  propagatedBuildInputs = [
    aiohttp
  ];
  
  checkInputs = [
    asynctest
    ddt
    pytest_3
  ];
```

`setup_requires` are usually `nativeBuildInputs` because they're a build-time dep.
Any dep that's needed for the checkPhase should be a `checkInputs`. 
They get added to `nativeBuildInputs` when `doCheck = true;` (default in buildPython*)

And we must propagate `aiohttp` because it's needed at runtime.

---

Aah great. That's better.